hashview/setup/__init__.py

```diff
@@ -0,0 +1,141 @@
+import os
+
+from pathlib import Path
+
+from flask_bcrypt import Bcrypt
+from flask_sqlalchemy import SQLAlchemy
+
+from hashview.models import Rules
+from hashview.models import Tasks
+from hashview.models import Users
+from hashview.models import Settings
+from hashview.models import Wordlists
+from hashview.utils.utils import get_filehash
+from hashview.utils.utils import get_linecount
+
+
+DEFAULT_PASSWORD = 'hashview'
+
+
+def default_tasks_need_added(db :SQLAlchemy) -> bool:
+    return (0 == db.session.query(Tasks).count())
+
+
+def add_default_tasks(db :SQLAlchemy):
+    task = Tasks(
+        name          = 'Rockyou Wordlist',
+        owner_id      = '1',
+        wl_id         = '1',
```

Comment:
This is correct for now, in so much as the wl_id=1 is rock you, but if we swap it so that the dynamic wordlist is loaded first, then these should be updated wl_id=2